internal/console/helpers/helpers_test.go

```diff
@@ -219,3 +220,64 @@ func TestProgressBar(t *testing.T) {
 		})
 	}
 }
+
+func TestFileAnalyzer(t *testing.T) {
+	if err := test.ChangeCurrentDir("kics"); err != nil {
+		t.Fatal(err)
+	}
+
+	tests := []struct {
+		name string
+		arg  string
+		want string
+	}{
+		{
+			name: "file_analizer_json",
+			arg:  "test/fixtures/config_test/kics.json",
+			want: "json",
+		},
+		{
+			name: "file_analizer_json_no_extension",
+			arg:  "test/fixtures/config_test/kics.config_json",
+			want: "json",
+		},
+		{
+			name: "file_analizer_yaml",
+			arg:  "test/fixtures/config_test/kics.yaml",
+			want: "yaml",
+		},
+		{
+			name: "file_analizer_yaml_no_extension",
+			arg:  "test/fixtures/config_test/kics.config_yaml",
+			want: "yaml",
+		},
+		{
+			name: "file_analizer_hcl",
+			arg:  "test/fixtures/config_test/kics.hcl",
+			want: "hcl",
+		},
+		{
+			name: "file_analizer_hcl_no_extension",
+			arg:  "test/fixtures/config_test/kics.config_hcl",
+			want: "hcl",
+		},
+		{
+			name: "file_analizer_toml",
+			arg:  "test/fixtures/config_test/kics.toml",
+			want: "toml",
+		},
+		{
+			name: "file_analizer_toml_no_extension",
+			arg:  "test/fixtures/config_test/kics.config_toml",
+			want: "toml",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := FileAnalyzer(tt.arg); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("FileAnalyzer() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
```

Comment:
Need to add a negative test: what if the content is neither of the supported formats? What is the behaviour?